src/app/components/Embeds/OEmbed/index.tsx

```diff
@@ -6,19 +6,21 @@ import { RequestContext } from '../../../contexts/RequestContext';
 import EmbedHtml from '../EmbedHtml';
 import EmbedError from '../EmbedError';
 
+type OEmbedData = Partial<{
```

Comment:
What does Partial do here?

Reply:
It's essentially the same as having all attributes set to optional, which basically represents a superset of those set of attributes: 

```
Example: 

Partial<{
    attr1: string;
    attr2: num;
}

= 

{
    attr1?: string;
    attr2?: string;
}
```

Reply:
From this Spike I didn't get the impression it was possible. (There is a slack thread linked [in this section ](https://paper.dropbox.com/doc/WSTEAM1-535-VJ-FABL-Spike-WS--CB~uFgcbXld_qC9E5hnv1PvEAg-2Dc47Qm0NobKt5izvMDMi#:uid=208935402585363762493318&h2=Outstanding-Question---Determi).

Reply:
Thank you! Looks all good to me then